### PR TITLE
Improve GUI

### LINKED ISSUE #22: Beautiful user interface
As a user, I can toggle between light/dark theme, so that I can customize the app to my preferences.

### PR DESCRIPTION
**Pull request by darkvoid32**
GUI follows legacy AB3 mockup

Improvements made to suit AssetBook needs

Closes #22, #118 and #163

### CODE REVIEW

File: src/main/java/seedu/address/ui/MainWindow.java [hunk 1]
```diff
@@ -125,6 +132,10 @@ private void fillInnerParts() {
 
         CommandBox commandBox = new CommandBox(this::executeCommand);
         commandBoxPlaceholder.getChildren().add(commandBox.getRoot());
+
+        File iconFile = new File(iconPath);
+        Image iconImage = new Image(iconFile.toURI().toString());
```
Comment (bryanyee33) [inline src/main/java/seedu/address/ui/MainWindow.java:137]: Would it be better to make this `Image iconImage = new Image(iconPath);` instead? This should work with my above requested change.

File: src/main/resources/view/MainWindow.fxml [hunk 2]
```diff
@@ -6,15 +6,17 @@
...
 <?import javafx.scene.image.Image?>
 <?import javafx.scene.layout.StackPane?>
 <?import javafx.scene.layout.VBox?>
 
+<?import javafx.scene.layout.HBox?>
+<?import javafx.scene.control.Label?>
+<?import javafx.scene.image.ImageView?>
 <fx:root type="javafx.stage.Stage" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1"
-         title="AssetBook-3" minWidth="450" minHeight="600" onCloseRequest="#handleExit">
+         title="AssetBook" minWidth="600" minHeight="800" onCloseRequest="#handleExit">
```
Comment (bryanyee33) [inline src/main/resources/view/MainWindow.fxml:17]: Would it be better to change minHeight to 700? The minHeight of 800 causes the app to go out of my screen, and forces the window to be unable to become smaller.

File: src/main/resources/view/MainWindow.fxml [hunk 3]
```diff
@@ -46,12 +63,11 @@
           </padding>
         </StackPane>
 
-        <VBox fx:id="personList" styleClass="pane-with-border" minWidth="340" prefWidth="340" VBox.vgrow="ALWAYS">
+        <StackPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="pane-with-border">
           <padding>
-            <Insets top="10" right="10" bottom="10" left="10" />
+            <Insets top="5" right="10" bottom="5" left="10" />
```
Comment (bryanyee33) [inline src/main/resources/view/MainWindow.fxml:68]: Should the left and right inset be 0 instead? It looks better if it fills the whole width. Same issue with the above resultDisplayPlaceholder.

**Review by bryanyee33 — CHANGES_REQUESTED** [review]
LGTM, just some minor nits.

File: src/main/java/seedu/address/ui/MainWindow.java (hunk 1 above)
Comment (darkvoid32) [reply src/main/java/seedu/address/ui/MainWindow.java]: File object removed

File: src/main/resources/view/MainWindow.fxml (hunk 2 above)
Comment (darkvoid32) [reply src/main/resources/view/MainWindow.fxml]: Min width and height reverted

File: src/main/resources/view/MainWindow.fxml (hunk 3 above)
Comment (darkvoid32) [reply src/main/resources/view/MainWindow.fxml]: Padding removed so everything fills up the width

**Review by bryanyee33 — APPROVED** [review]
LGTM. Nice work on the icons!

**Pull request merged**